src/Components/HourlyWeather/index.js

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import "./index.css";
+
+const HourlyWeather = ({ placename }) => {
+    const [hourlyData, setHourlyData] = useState([]);
+
+    useEffect(async () => { 
+        async function convertGeoLocation(placename) {
+            const response = await fetch(`https://api.opencagedata.com/geocode/v1/json?q=${placename}&key=${process.env.OPEN_CAGE_APIKEY}`).then(value => value.json());
+            const cord = response["results"][0]["geometry"];
+            return [cord["lat"], cord["lng"]];
```

Comment:
Add Exceptional handling

In case the API call fails, it should show a nice error message.

Add this in a different PR but mentioning it here so you know it.